cmd/kube-apiserver/app/server.go

```diff
@@ -319,6 +321,15 @@ func CreateKubeAPIServerConfig(s *options.ServerRunOptions, nodeTunneler tunnele
 		return nil, nil, nil, nil, nil, err
 	}
 
+	var issuer serviceaccount.TokenGenerator
+	if s.IDTokenIssuerKey != "" && s.IDTokenIssuerID != "" {
```

Comment:
Should we error if these are provided but the feature gate isn't enabled?

Reply:
Done.

Reply:
error if one is provided and not the other